jmbitcoin/setup.py

```diff
@@ -10,5 +10,5 @@
       license='GPL',
       packages=['jmbitcoin'],
       python_requires='>=3.6',
-      install_requires=['python-bitcointx>=1.1.1.post0', 'pyaes', 'urldecode'],
+      install_requires=['python-bitcointx>=1.1.3', 'pyaes', 'urldecode'],
```

Comment:
Pin to a specific version?
```suggestion
      install_requires=['python-bitcointx==1.1.3', 'pyaes', 'urldecode'],
```

Reply:
Yeah you're right. Changing.

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1181/commits/a467ec460e15be3b626d64e13c3543f12fd24b91